README.md

```diff
@@ -66,34 +66,40 @@ Install the ``python3-routes`` package:
 
 These instructions are meant for developers who are compiling the code for
 development and testing. To build binaries that are suitable for installation
-we recommend that you build .deb or .rpm packages, or refer to ``ceph.spec.in``
-or ``debian/rules`` to see which configuration options are specified for
-production builds.
+we recommend that you build `.deb` or `.rpm` packages, or refer to
+``ceph.spec.in`` or ``debian/rules`` to see which configuration options are
+specified for production builds.
```

Comment:
This is something that has come up on the list in the past:  Here we're saying to build packages, but for operators we push people to use `cephadm` and containers.  When asked how Joe Cephpack can build the containers, the only answer I've ever seen is "It's complicated".

Reply:
I'm working on a good explanation of this fit for user consumption. It'll take about a month, and you'll be along for every step of it.